.github/workflows/core-conda-bld.yml

```diff
@@ -0,0 +1,48 @@
+name: test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test-w-conda-recipe:
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: checkout
```

Comment:
A very minor point, but I'd skip naming steps that are already described by their `uses` names.